commcare_connect/static/prelogin/app.js

```
@@ -91,109 +91,22 @@ function routeFromPath(pathname) {
-  },
-  '/support-kmc': {
-    title: 'Fund a Frontline Worker | Connect by Dimagi',
-    desc: 'Your gift funds a trained Frontline Worker to deliver verified Kangaroo Mother Care home visits to small and vulnerable newborns. $60 covers one complete, verified intervention.',
-  },
-};
+const ROUTE_META = (function () {
+  const node = document.getElementById('route-meta');
+  if (!node) return {};
+  try {
+    return JSON.parse(node.textContent);
+  } catch (e) {
```

ESLint

[error] 'e' is defined but never used. Allowed unused caught errors must match /^_$/u.

(no-unused-vars)